torch/_dynamo/variables/torch.py

```diff
@@ -1242,14 +1245,15 @@ def speculate_branch(branch):
         else:
             unimplemented(f"HigherOrderOperator {self.value.__name__}")
 
+        p_kwargs = {k: v.as_proxy() for k, v in kwargs.items()}
```

Comment:
This is a less safe proxy_args_kwargs. As you have p_args done, just do p_kwargs/

```suggestion
        p_kwargs = proxy_args_kwargs([], kwargs) 
```